packages/design-system/src/components/web-components/ds-month-picker/ds-month-picker.stories.tsx

```diff
@@ -0,0 +1,147 @@
+      description: 'Set to "true" to apply the "inverse" scheme',
+      control: 'boolean',
+    },
+    'label-class-name': {
+      description: 'Additional classes to be added to the field label',
+      control: 'text',
+    },
+    'label-id': {
+      description:
+        "A unique `id` to be used on the field label. If one isn't provided, a unique ID will be generated.",
+      control: 'text',
+    },
+    label: {
+      description: 'Label text or HTML.',
+      control: 'text',
+    },
+    name: {
+      description: "The `input` field's `name` attribute.",
+      control: 'text',
+    },
+    'requirement-label': { control: 'text' },
+    'root-id': {
+      description:
+        "A unique ID for this element. A unique ID will be generated if one isn't provided.",
+      control: 'text',
+    },
+    'select-all-text': {
+      description: 'The text for the "Select all" button for internationalization.',
+      control: 'text',
+    },
+  },
+  args: {
+    label: 'Month Picker example',
+    'requirement-label': 'Required.',
+    hint: 'Culpa minim deserunt anim consequat amet sit.',
+    name: 'month-picker_field',
+    children: options,
+  },
+  parameters: {
+    docs: {
+      page: WebComponentDocTemplate,
+      description: {
+        component:
+          'For information about how and when to use this component, [refer to its full documentation page](https://design.cms.gov/components/month-picker/).',
+      },
+      componentEvents: {
+        'ds-change': {
+          description:
+            'A callback function that\'s invoked when a month\'s checked state is changed. Note: This callback is not called when a month is selected or deselected via the "Select all" or "Clear all" buttons – use the `onSelectAll` and `onClearAll` event handlers for those instances.',
+          eventObjectDescription: (
+            <>
+              <code>event.details.target.value</code> - The <code>value</code> of the selected
+              option
+            </>
+          ),
+        },
+      },
+    },
+  },
+  decorators: [webComponentDecorator],
+};
+
+const Template = (args) => {
+  useEffect(() => {
+    const onChange = (event) => {
+      action('ds-change')(event);
+    };
+    const onClearAll = (event) => {
+      action('ds-clear-all')(event);
+    };
+    const onSelectAll = (event) => {
+      action('ds-select-all')(event);
+    };
+    const monthPicker = document.querySelector('ds-month-picker');
+    monthPicker.addEventListener('ds-change', onChange);
+    monthPicker.addEventListener('ds-clear-all', onClearAll);
+    monthPicker.addEventListener('ds-select-all', onSelectAll);
+    return () => {
+      monthPicker.removeEventListener('ds-change', onChange);
+      monthPicker.removeEventListener('ds-clear-all', onClearAll);
+      monthPicker.removeEventListener('ds-select-all', onSelectAll);
+    };
+  });
+
+  return <ds-month-picker {...args} />;
+};
+
+export const Default = Template.bind({});
```

Comment:
The default story on at the bottom of the docs page actually fails to render for me unless I make the following change to the decorator component in `web-components/storybook.tsx` to strip out the Preact element children:

```tsx
export function webComponentDecorator(Story, context) {
  // The Preact element `children` can have circular references which trip up Storybook's JSON evaluation
  const {children, ...simpleArgs} = context.args;
  return (
    <Fragment key={JSON.stringify({ ...simpleArgs, ...context.globals })}>
      <Story />
    </Fragment>
  );
}
```